xlive/H2MOD/Discord/DiscordInterface.cpp

```diff
@@ -76,6 +84,35 @@ static void updateDiscordPresence()
 	discordPresence.largeImageText = map_mouse_over;
 	discordPresence.smallImageKey = mode_key;
 	discordPresence.smallImageText = mode_details;
+	if (b_inGame != false) //called only in MP engine
+	{
+		previousRoundNumber = *(int*)((BYTE*)0x300056C6);
```

Comment:
please document what this random offset is (move the value into a constant value as well)

Reply:
preferably move them into functions that are readable